usim/_primitives/activity.py

```diff
@@ -51,16 +51,37 @@ def __transcript__(self) -> ActivityCancelled:
 
 
 class ActivityExit(BaseException):
-    ...
+    """A :py:class:`~.Activity` forcefully exited"""
 
 
-class Activity(Condition, Generic[RT]):
+class Activity(Awaitable[RT]):
     """
-    Active coroutine that allows others to listen for its completion
+    Concurrently running activity that allows multiple activities to await its completion
 
-    :note: Simulation code should never instantiate this class directly.
+    A :py:class:`Activity` represents an activity that is concurrently run in a :py:class:`~.Scope`.
```

Comment:
Maybe some wording issues:

* "A :py:class:`Activity`" -> An :py:class:`Activity`
* "that is concurrently run" -> that runs concurrently (not sure about this one...)